src/sas/sascalc/dataloader/readers/IgorReader.py

```diff
@@ -113,14 +113,14 @@ def read(self, filename=None):
                 # Get total bin number
                 
             i_tot_row += 1
-        i_tot_row = math.ceil(math.sqrt(i_tot_row)) - 1
-        #print "i_tot", i_tot_row
+        i_tot_row = int(math.ceil(math.sqrt(i_tot_row)) - 1)
+        # print "i_tot", i_tot_row

```

Comment:
Estimating the size of the detector array by taking the sqrt of the total number of lines in the file is 'interesting'. Perhaps a better way is to figure out when the data starts. THe total number of lines is 16403, the number of header lines is 19. So the size of each side of the data array must be sqrt(16403 - 19) = 128.